services/pane.js

```diff
@@ -91,36 +90,40 @@ function createPublishMessages(res) {
  * @returns {Element}
  */
 function createPublishActions(res) {
-  var today = moment().format('YYYY-MM-DD'),
+  var actions = dom.find('.publish-actions'),
+    today = moment().format('YYYY-MM-DD'),
     now = moment().format('HH:mm'),
-    actions = dom.create('<div class="actions"></div>'),
-    schedule = dom.create(`<form class="schedule">
-      <label class="schedule-label" for="schedule-date">Date</label>
-      <input id="schedule-date" class="schedule-input" type="date" min="${today}" value="${today}" placeholder="${today}"></input>
-      <label class="schedule-label" for="schedule-time">Time</label>
-      <input id="schedule-time" class="schedule-input" type="time" value="${now}" placeholder="${now}"></input>
-      <button class="schedule-publish">Schedule Publish</button>
-    </form>`),
-    unschedule = dom.create('<button class="unschedule">Unschedule</button>'),
-    publishNow = dom.create('<button class="publish-now">Publish Now</button>'),
-    unpublish = dom.create('<button class="unpublish">Unpublish Page</button>');
-
-  // add actions and buttons (in order)
-  // scheduling: always exists
-  actions.appendChild(schedule);
+    scheduleDate, scheduleTime, unpublish, unschedule;
+
+  // set date and time
+  if (actions) {
+    scheduleDate = dom.find(actions, '#schedule-date');
+    scheduleTime = dom.find(actions, '#schedule-time');
+
+    if (scheduleDate) {
+      dom.replaceElement(scheduleDate, dom.create(`<input id="schedule-date" class="schedule-input" type="date" min="${today}" value="${today}" placeholder="${today}"></input>`));
+    }
+
+    if (scheduleTime) {
+      dom.replaceElement(scheduleTime, dom.create(`<input id="schedule-time" class="schedule-input" type="time" value="${now}" placeholder="${now}"></input>`));
```

Comment:
wait, aren't these already in the dom? Why are we replacing elements rather than just updating their attributes?